### PR TITLE
Azamat Valitov - Feed API Challenge

### PR DESCRIPTION
**Pull request by valitovaza**

### CODE REVIEW

File: FeedAPIChallenge/Feed API/RemoteFeedLoader.swift [hunk 1]
```diff
@@ -19,6 +19,41 @@ public final class RemoteFeedLoader: FeedLoader {
...
+			return .success(root.items.map({$0.feedImage}))
+		}else{
+			return .failure(Error.invalidData)
+		}
+	}
+}
+private struct Root: Decodable {
+	let items: [Item]
+}
+private struct Item: Decodable {
```
Comment (mapostolakis) [inline FeedAPIChallenge/Feed API/RemoteFeedLoader.swift:50]: The repo convention dictates leaving a space before and after a brace, e.g., `} else {` and a line break before declaring a new function or type, e.g.,

```swift
private var validStatusCode: Int { 200 }

private func convert(_ data: Data) -> FeedLoader.Result {
```

Please update any occurrences in the code to follow the convention.

**Review by mapostolakis — COMMENTED** [review]
Hi @valitovaza, you almost passed the challenge!

Please, check my comments and let me know once you're done so I can review it again. Thank you 🙂

File: FeedAPIChallenge/Feed API/RemoteFeedLoader.swift (hunk 1 above)
Comment (valitovaza) [reply FeedAPIChallenge/Feed API/RemoteFeedLoader.swift]: Hey Mike, thank you for the review. I made changes, please take a look!

**Review by mapostolakis — COMMENTED** [review]
Hi @valitovaza,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs.

Great job! ✅

**Pull request closed without merging**